internal/plugin/v1/connection_manager.go

```diff
@@ -8,26 +8,14 @@ import (
 
 // ConnectionManager is an interface to be implemented by plugins that want to
 // manage connections for handlers and listeners.
+// TODO: Add back methods for connector-based events
```

Comment:
this will show in the godocs - can we record this another way? strong preference to a github issue over polluting our godocs with a bunch of TODOs